### PR TITLE
Bump gimli to 0.18.0

### PR DESCRIPTION
**Pull request by bjorn3**
Also adds travis config.

### CODE REVIEW

File: unwind/src/lib.rs [hunk 1]
```diff
@@ -96,28 +96,26 @@ struct UnwindInfo<R: Reader> {
...
-            eh_frame: ref sel,
+            ref eh_frame,
             ref bases,
             ..
         } = self;
 
         let fde = eh_frame_hdr.table().unwrap()
-            .lookup_and_parse(address, bases, sel.clone(), |offset| sel.cie_from_offset(bases, offset))?;
-
+            .fde_for_address(eh_frame, bases, address, |eh_frame, bases, offset| eh_frame.cie_from_offset(bases, offset))?;
```
Comment (philipc) [inline unwind/src/lib.rs]:
```suggestion
            .fde_for_address(eh_frame, bases, address, EhFrame::cie_from_offset)?;
```

Comment (bjorn3) [reply unwind/src/lib.rs]: Didn't notice the signatures were the same :)

**Comment by philipc** [comment]
Hmm, this doesn't run the tests in travis, and the tests seem broken.

**Comment by bjorn3** [comment]
I dont think travis is setup.

**Comment by philipc** [comment]
It is, see #33 for example.

**Comment by bjorn3** [comment]
Oops, I looked at travis-ci.org instead of travis-ci.com.

**Pull request merged**